src/networking.c

```diff
@@ -3190,5 +3202,10 @@ int handleClientsWithPendingReadsUsingThreads(void) {
         }
         processInputBuffer(c);
     }
+
+     /* Update processed count on server */
+    server.stat_threaded_read_processed += processed-processed_by_main_thread;
+    server.stat_main_thread_read_processed += processed_by_main_thread;
```

Comment:
When I mentioned the main thread/background threads, the information I thought was useful was trying to understand if an application was effectively using the threaded reads. The current implementation would just show the percentage of threaded IO performed by the main thread, which is ultimately not that useful. 

So I was thinking of having one variable for reads processed the regular way and one processed the old way:
1. threaded_io_reads_processed (main thread + IO threads)
2. single_threaded_reads_processed (Main thread the old way)

Reply:
I think i understand what you are looking for. I made the following changes to adapt these changes:
- I added count for all reads 
- I added count for thread reads
And to get the number of of `single_threaded_reads_processed`, I subtract the total reads from thread reads.

Is this sufficient? Let me know If I am missing something.

Thanks.